README.md

```diff
@@ -1,2 +1,90 @@
 # Capstone-project
 capston project website
+## html and styles css & javascript
+
+- english language
+- visual code editor
+- html , css, javascript
+
+## Live Demo (if available)
+
```

Comment:
- [ ] It is mandatory in this project to add a link of a deployed application.  You should use github pages to deploy your web pages. Kindly work on it.

Reply:
kindly, could you make me remember about deployment application , actually i dont remember

Reply:
- [ ] About deployment, this [link](https://www.codecademy.com/article/f1-u3-github-pages) could be helpful.